boards/arm/nrf52810_pca20045/Kconfig

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+if BOARD_NRF52810_PCA20045
+
+config BOARD_ENABLE_DCDC
+        bool "Enable DCDC mode"
+        select SOC_DCDC_NRF52X
+        default y
+
+# BT_CTLR depends on BT. When BT is enabled we should default to also
+# enabling the controller.
+config BT_CTLR
+	default y if BT
```

Comment:
For almost all boards upstream (nrf52810_pca10040 is probably the only exception) it is done in `Kconfig.defconfig` as:
```
config BT_CTLR
	default BT
```
Let's keep this pattern here.

Reply:
done